src/Symfony/Bundle/FrameworkBundle/Command/DebugDeprecationsWarmedCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Show the deprecations generated during the cache warmup.
+ *
+ * @author Hamza Amrouche <hamza.simperfit@gmail.com>
+ *
+ * @final
+ */
+class DebugDeprecationsWarmedCommand extends Command
+{
+    protected static $defaultName = 'debug:deprecations:warmed';
+
+    private $containerDeprecationFilePath;
+
+    public function __construct(string $containerDeprecationFilePath)
+    {
+        parent::__construct();
+
+        $this->containerDeprecationFilePath = $containerDeprecationFilePath;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Show the deprecated calls generated during cache:warmup')
+            ->setHelp(<<<'EOF'
+The <info>%command.name%</info> shows depecrated calls generated during cache:warmup
+
+  <info>php %command.full_name%</info>
+EOF
+            )
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+        if (!file_exists($this->containerDeprecationFilePath)) {
+            $io->warning('The deprecation file does not exist, please try warming the cache before.');
+
+            return;
+        }
+
+        $logs = unserialize(file_get_contents($this->containerDeprecationFilePath));
+        if (0 === \count($logs)) {
+            $io->success('There are no more deprecations in the logs!');
+
+            return;
+        }
+
+        $formattedLogs = [];
+        $remainingCount = 0;
+        foreach ($logs as $log) {
+            $formattedLogs[] = sprintf("%sx: %s \n in %s:%s ", $log['count'], $log['message'], $log['file'], $log['line']);
```

Comment:
could you please add an indent space after the breakline?

Reply:
done